src/Document.php

```diff
@@ -675,26 +694,43 @@ public function canViewFile(array $options = [])
             return true;
         }
 
+        // new options
+        $itemtype = $options['itemtype'] ?? null;
+        $items_id = $options['items_id'] ?? null;
+
+        // legacy options
+        $changes_id  = $options['changes_id'] ?? null;
```

Comment:
```suggestion
        $changes_id  = $options['changes_id'] ?? ($itemtype === "Change" ? $items_id : null);
```

Maybe we can do something like this to avoid complex if statements below, no

Reply:
I pushed a commit for this. I choose to give priority to new options.